exploration/expression-attributes.md

```diff
@@ -133,7 +175,38 @@ esp. if a similar expression is used in multiple variants.
 
 Comments should not influence the runtime behaviour of a formatter.
 
-### Define `@attributes` as above, but do not namespace custom attributes
+### Define `@attributes` as above, but explicitly namespace custom attributes
+
+As namespacing may also be required for function names and function option names,
+and because we want to allow at least for custom function options
+to be definable on default formatters,
+the namespace rules for parts of the specification would end up differing.
+
+By suggesting instead of requiring,
+we rely on our stability policy to guide implementations to keep clear of the namespace
+that may be claimed by later versions of the specification.
+
+### Enable function chaining within a single expression
+
+By allowing for multiple annotation functions on a single expression,
+it becomes possible to consider some of them to have passthrough behaviour during formatting
+but have an impact during translation.
+
+For example, with an expression like
+
+```
+{42 :locale set=fr :number :xliff can-copy=yes}
+```
+
+the locale may be set separately from other options,
+and a custom attribute applied during a conversion to XLIFF.
+
+This approach may produce the same result as the proposed design,
+but has a few caveats:
 
-Later spec versions will not be able to define _any_ new attributes
-without a danger of breaking implementations or users already using those names.
+- The order of operations matters,
+  and an attribute such as the locale must be set before others that are influenced by it.
```

Comment:
There is the order of evaluation to consider. I was asking myself if the above example evaluated from right-to-left (since setting the locale after formatting a number would be backwards)

Reply:
```suggestion
- The order of operations matters,
  and an attribute such as the locale must be set before others that are influenced by it.
- It's not clear whether the functions will be evaluated
  left-to-right or right-to-left.
```